src/transformers/models/vit/modeling_tf_vit.py

```diff
@@ -662,7 +662,6 @@ def call(
         Returns:
 
         Examples:
-
```

Comment:
I wasn't sure if there is supposed to be an empty line here or not, for me it looked better without it

Reply:
Let's just keep the empty line. It is done this way in other places.